.drone.yml

```diff
@@ -0,0 +1,85 @@
+workspace:
+  base: /go
+  path: src/github.com/src-d/gitbase-playground
+
+branches: [master]
+
+# ANCHORS
+
+build: &build
+  image: golang:1.10-stretch
+  commands:
+    - curl -sL https://deb.nodesource.com/setup_8.x | bash
+    - apt-get update
+    - apt-get install -y --no-install-recommends --no-install-suggests nodejs libxml2-dev
+    - npm install -g yarn
+    - make dependencies
+    - PKG_OS=linux DOCKER_OS=linux DOCKER_ARCH=amd64 make packages
+  debug: true
+
+docker_image: &docker_image
+  group: docker
+  image: plugins/docker
+  registry: docker.io
+  repo: srcd/gitbase-playground
+  secrets: [ docker_username, docker_password ]
+  dockerfile: Dockerfile
+  debug: true
+
+helm_deploy: &helm_deploy
+  image: quay.io/ipedrazas/drone-helm:master-9b37211
+  skip_tls_verify: true
+  helm_repos: charts=https://github.com/src-d/charts.git
```

Comment:
I don't think a git repository is what `drone-helm` is expecting, judging from https://github.com/ipedrazas/drone-helm#using-private-repositories documentation

I'd test with something like

```
helm_repos: srcd-charts=https://src-d.github.io/charts/
chart: srcd-charts/gitbase-playground
```
also setting a different name to avoid using `charts`, that seems quite generic

Reply:
thanks, fixed in 81194b3.